packages/server/src/mapping/convmap/service.ts

```diff
@@ -55,7 +56,8 @@ export class ConvmapService extends Service {
 
     await this.batcher.push(convmap)
     this.cacheByThreadId.set(tunnelId, threadId, convmap)
-    this.cacheByConversationId.set(tunnelId, conversationId, convmap)
+    // TODO: What to do with this? This could lead to errors
```

Comment:
What do you mean? Is this wrong? If so, change TODO to FIXME

Reply:
It's a problem that could happen in the future but not at the moment.